markdown/events/2022/hackathon-march-2022.md

```diff
@@ -11,7 +11,10 @@ import_typeform: true
 ---
 
 
-Please join us for the March 2022 nf-core hackathon!
+Please join us for the March 2022 nf-core hackathon! 
+
+After all the fun changes introduced with the DSL2, the focus of this hackathon will be all things documentation. This will include documentation for pipelines and modules, trainings and tutorials, nf-core tools etc.
```

Comment:
```suggestion
### Topic: Documentation

After all the fun changes introduced with the DSL2, the focus of this hackathon will be all things documentation. This will include documentation for pipelines and modules, trainings and tutorials, nf-core tools etc.
```